src/EventStore.Client/EventStoreClientBase.cs

```diff
@@ -85,10 +86,13 @@ public abstract class EventStoreClientBase :
 			return new(channel, caps, invoker);
 		}
 
-#pragma warning disable 1591
+
+		/// <summary>
+		/// Gets the current channel info in an inexpensive way.
```

Comment:
```suggestion
		/// Gets the current channel info.
```

(it's not necessarily inexpensive, it _might_ need to do discovery)